src/SDKs/TrafficManager/Management.TrafficManager/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.2.0-preview",
+  "version": "2.3.0",
```

Comment:
@vladca all traces of project.json, global.json and *.xproj should be deleted. These files are no needed anymore in the new project system.